src/functionalTest/java/uk/gov/hmcts/dts/fact/CourtsEndpointTest.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.dts.fact;
+
+import io.restassured.RestAssured;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpHeaders.CONTENT_TYPE;
+
+@ExtendWith(SpringExtension.class)
+public class CourtsEndpointTest {
+
+    private static final String AYLESBURY_MAGISTRATES_COURT_AND_FAMILY_COURT
+        = "aylesbury-magistrates-court-and-family-court";
+
+    @Value("${TEST_URL:http://localhost:8080}")
```

Comment:
As this grows I expect we'll probably move some of the config out into a yaml file for the functionalTest package

Reply:
Agreed.